### DISCUSSION TITLE
The compiler is added as dependency instead of peer dependency

### DISCUSSION THREAD

**Discussion by bdm-k** [microsoft/typespec#3625] [Q&A]
When you create a TypeSpec emitter with the template, `@typespec/compiler` is added as a dependency instead of a peer dependency. However, according to the [document](https://typespec.io/docs/extending-typespec/basics#step-3-defining-dependencies), you should

> Use `peerDependencies` for all TypeSpec libraries (and the compiler) that you use in your own library or emitter.

Is this intended behavior or not?

### How to reproduce

1. Install `@typespec/compiler@0.57.0`
2. Run `tsp init --template emitter-ts`

In the generated "package.json", `@typespec/compiler` is added as a dependency.

**Comment by bterlson** [comment]
Good catch!

**Comment by bdm-k** [comment]
This is a bug, and a corresponding issue has been created. #3632
